### PR TITLE
Use success instead of stopped when no failure

### LINKED ISSUE #8191: Rename STOPPED to SUCCESS
I find this constant to be ambigous, as "stopped" as a word could indicate some kind of failure, but it is always used when the ensemble has finished without errors. We call the corresponding thing for a realization for "success" so we should probably use that word instead.

https://github.com/equinor/ert/blob/3f3e682156516d15ccded659adefb3730da43e83/src/ert/event_type_constants.py#L10

### PR DESCRIPTION
**Pull request by berland**
The word STOPPED is too ambigous, use SUCCESS instead for ensembles that go through without failure. This is in line with event names for realizations.

**Issue**
Resolves #8191 

**Approach**
do-it

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/ensemble_evaluator/evaluator.py [hunk 1]
```diff
@@ -302,7 +302,7 @@ async def handle_dispatch(
                         return
 
                     if event["type"] in [
-                        EVTYPE_ENSEMBLE_STOPPED,
+                        EVTYPE_ENSEMBLE_SUCCESS,
                         EVTYPE_ENSEMBLE_FAILED,
```
Comment (jonathan-eq) [inline src/ert/ensemble_evaluator/evaluator.py:306]: Should we rename the `EVTYPE_ENSEMBLE_SUCCESS` to `EVTYPE_ENSEMBLE_SUCCEEDED` or the failed event type to `EVTYPE_ENSEMBLE_FAILURE`?

**Review by jonathan-eq — CHANGES_REQUESTED** [review]
This would help readability :)
See minor changes

**Comment by berland** [comment]
Fixed (and more)!

File: src/ert/ensemble_evaluator/evaluator.py [hunk 2]
```diff
@@ -302,7 +302,7 @@ async def handle_dispatch(
                         return
 
                     if event["type"] in [
-                        EVTYPE_ENSEMBLE_STOPPED,
+                        EVTYPE_ENSEMBLE_SUCCESS,
```
Comment (jonathan-eq) [inline src/ert/ensemble_evaluator/evaluator.py:305]: Should we rename the EVTYPE_ENSEMBLE_SUCCESS to EVTYPE_ENSEMBLE_SUCCEEDED or the failed event type to EVTYPE_ENSEMBLE_FAILURE

Comment (berland) [reply src/ert/ensemble_evaluator/evaluator.py:305]: Changing to SUCCEEDED.

Comment (berland) [reply src/ert/ensemble_evaluator/evaluator.py:293]: Changing SUCCESS to SUCCEEDED.

**Review by jonathan-eq — APPROVED** [review]
LGTM! :ship:

**Pull request merged**